packages/react-devtools-extensions/flow-typed/npm/react-test-renderer_v16.x.x.js

```diff
@@ -77,7 +79,7 @@ declare module 'react-test-renderer/shallow' {
     static createRenderer(): ShallowRenderer;
     getMountedInstance(): ReactTestInstance;
     getRenderOutput<E: React$Element<any>>(): E;
-    getRenderOutput(): React$Element<any>;
+    getRenderOutput(): React$Element<any>; // eslint-disable-line ft-flow/no-dupe-keys
```

Comment:
What is this for? can it be deleted?

Reply:
I thought it was method overloading, but looking at it again that doesn't seem to be the case. I'll try if flow passes when I remove this.